src/app/features/animals/components/animals-list/animals-list.component.html

```diff
@@ -1,4 +1,60 @@
-<h1>Animals</h1>
+<div class="row">
+  <div class="col">
+    <h1>Animals</h1>
+  </div>
+  <div class="col text-end">
+    <button class="btn btn-success" routerLink="/animals/add">Add new</button>
+  </div>
+</div>
 
-<!-- TODO: Add table with animals -->
-<p>Coming soon...</p>
+<ul class="nav nav-tabs mb-4">
+  <li class="nav-item">
+    <a class="nav-link" (click)="changeActiveTab('table')" [class.active]="activeTab === 'table'">Table</a>
+  </li>
+  <li class="nav-item">
+    <a class="nav-link" (click)="changeActiveTab('cards')" [class.active]="activeTab === 'cards'">Cards</a>
+  </li>
+</ul>
+
+<div>
+  <ng-container *ngIf="activeTab === 'table'">
+    <table class="table table-striped align-middle">
+      <thead>
+      <tr>
+        <th scope="col">#</th>
+        <th scope="col">Name</th>
+        <th scope="col">Type</th>
+        <th scope="col" class="text-end">Actions</th>
+      </tr>
+      </thead>
+      <tbody>
+      <tr *ngFor="let animal of animalsService.animals; let i = index">
```

Comment:
V angulari sa uz po novom inak pouziva ngFor, ngIf a pod. Viac info vies najst tu: [Angular is getting New Template Syntax](https://dev.to/this-is-angular/angular-is-getting-new-template-syntax-4naf)

```suggestion
      <tr *ngFor="let animal of animalsService.animals; let i = index">
      @for (animal of animalsService.animals; let i = index) 
{
  ...
}
```

Reply:
Okrem toho sa tu volá z template priamo `animalsService`, čo by sa nemalo robiť. Mal by na to byť nejaký getter, ktorý vracia `Observable` a následne by sa mala použiť `async` pipe. [Pozri async pipe](https://medium.com/@softwaretechsolution/async-pipe-in-angular-bf0c691faaf2)

```suggestion
      @for (animal of animals$ | async; track $index) {
      {
        ...
      }
```

Reply:
Nizsie mas tiez starsie formy zapisu tak to prosim prerob tiez.